src/main/java/Duke.java

```diff
@@ -1,10 +1,251 @@
+import java.util.Locale;
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int MAX_RECORDS = 100;
+    public static final String INDENT = "    │ ";
+    private static int taskCount = 0;
+    private static int taskCompleted = 0;
+    private static final Task list[] = new Task[MAX_RECORDS];
```

Comment:
Perhaps this name [could be more descriptive](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#use-nouns-for-things-and-verbs-for-actions)? There may be different lists in a program.